bin/yangGeneric.py

```diff
@@ -17,32 +17,102 @@
 import json
 import os
 import re
+import typing as t
 
 import requests
 from filelock import FileLock
 
 from create_config import create_config
+from compilation_status import combined_compilation, pyang_compilation_status
+from extract_emails import extract_email_string
 from file_hasher import FileHasher
 from filesGenerator import FilesGenerator
 from parsers.confdcParser import ConfdcParser
 from parsers.pyangParser import PyangParser
 from parsers.yangdumpProParser import YangdumpProParser
 from parsers.yanglintParser import YanglintParser
-from utility.utility import (check_yangcatalog_data, dict_to_list,
-                             list_br_html_addition, module_or_submodule,
+from utility.utility import (check_yangcatalog_data, module_or_submodule,
                              number_that_passed_compilation, push_to_redis)
-from versions import ValidatorsVersions
 
 __author__ = 'Benoit Claise'
-__copyright__ = 'Copyright(c) 2015-2018, Cisco Systems, Inc.,  Copyright The IETF Trust 2019, All Rights Reserved'
+__copyright__ = 'Copyright(c) 2015-2018, Cisco Systems, Inc.,  Copyright The IETF Trust 2022, All Rights Reserved'
 __license__ = 'Apache License, Version 2.0'
 __email__ = 'bclaise@cisco.com'
 
-# ----------------------------------------------------------------------
-# Validators versions
-# ----------------------------------------------------------------------
-validators_versions = ValidatorsVersions()
-versions = validators_versions.get_versions()
+
+# Classes for generating compilation result table rows and metadata to be uploaded to confd
+
+class MetadataGenerator:
```

Comment:
What about moving this Generators into separate files? Maybe also moved them into generators directory